htdocs/commande/card.php

```diff
@@ -2633,7 +2633,7 @@
 			$somethingshown = $form->showLinkedObjectBlock($object, $linktoelem, $compatibleImportElementsList);
 
 			// Show online payment link
-			$useonlinepayment = (!empty($conf->paypal->enabled) || !empty($conf->stripe->enabled) || !empty($conf->paybox->enabled));
+			$useonlinepayment = ((!empty($conf->paypal->enabled) || !empty($conf->stripe->enabled) || !empty($conf->paybox->enabled)) && !empty($conf->facture->enabled));
```

Comment:
If we don't use the invoice module, we still should be able to record payment.
Test 
if !empty($conf->facture->enabled)   must be done inside the  paymentok.php file and is the condition to know if we must auto-create the invoice and payment or not.